packages/common/src/hooks/useDocument/useDocument.test.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Dispatch, FC } from 'react';
+import {
+  DefaultDocumentAction,
+  TestingProvider,
+  TestingRouter,
+} from '@openmsupply-client/common';
+import { waitFor } from '@testing-library/dom';
+import { useDocument } from './useDocument';
+import { Api } from './types';
+import { renderHook } from '@testing-library/react-hooks';
+import { Route } from 'react-router';
+
+describe('useDocument', () => {
```

Comment:
these tests are pretty lame but I think show a fine example of a simplified version of using the hook